### DISCUSSION TITLE
Is there any way to prevent Vercel for github check "Git author must have access to the project on Vercel"

### DISCUSSION THREAD

**Discussion by anhdd-kuro** [vercel/vercel#8491] [Help] [answered]
My project is using Monorepo with "server" and "client"
The problem is only "client" is using vercel , and only client's members is joining project on Vercel 
When server's member commit , github ci return Error "Git author must have access to the project on Vercel"
Is there any way to prevent commit author check if there is nothing change in client ?
Seems like ignore building step run after author check so it doesn't work

Or maybe a way to approve that commit from who have access to the project ?

_p/s : I don't want to add all members in project on vercel because vercel's pricing is per member..._

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @anhdd-kuro! Based on your description, I'm guessing that you have a private monorepo. You can learn more about limitations around [deploying private repos](https://vercel.com/docs/concepts/git#deploying-private-git-repositories) using teams here: https://vercel.com/docs/concepts/git#using-teams

For a public repo, a link to authorize the Deployment would be posted as a comment on the Pull Request and a team member could approve it. The security feature that requires membership in this case is called [Git Fork Protection](https://vercel.com/changelog/git-fork-protection-can-now-be-disabled) and it can be disabled if you are sure it won't pose a security risk for your repo.

Customizing GitHub Actions might be an option for you, though I have not personally tried it for this kind of scenario. You can learn more about using GitHub Actions with Vercel here: https://vercel.com/guides/how-can-i-use-github-actions-with-vercel

**Reply by anhdd-kuro** [reply]
Thanks for reply !
I will try the Customizing GitHub Actions option !
